src/server/game/Entities/Player/Player.cpp

```diff
@@ -23566,6 +23566,24 @@ void Player::SetDailyQuestStatus(uint32 quest_id)
     }
 }
 
+bool Player::IsDailyQuestDone(uint32 quest_id)
+{
+    bool found = false;
+    if (sObjectMgr->GetQuestTemplate(quest_id))
+    {
+        for (uint32 quest_daily_idx = 0; quest_daily_idx < PLAYER_MAX_DAILY_QUESTS; ++quest_daily_idx)
+        {
+            if (GetUInt32Value(PLAYER_FIELD_DAILY_QUESTS_1 + quest_daily_idx) == quest_id)
+            {
+                found = true;
+                break;
+            }
+        }
+    }
+
+    return found;
+}
+
```

Comment:
you could avoid `break` and put `return true`, and at the end `return false`, in this way you can remove the variable `found`.
I mean, in this way:
```C++
bool Player::IsDailyQuestDone(uint32 quest_id)
{
    if (sObjectMgr->GetQuestTemplate(quest_id)) {
        for (uint32 quest_daily_idx = 0; quest_daily_idx < PLAYER_MAX_DAILY_QUESTS; ++quest_daily_idx) {
            if (GetUInt32Value(PLAYER_FIELD_DAILY_QUESTS_1 + quest_daily_idx) == quest_id) {
              return true;
            }
        }
    }

    return false;
}
```